apps/dev-api/domains/common/utils/serverClients.js

```diff
@@ -0,0 +1,60 @@
+const get = require('lodash/get')
+
+const { ApolloServerClient } = require('@open-condo/apollo-server-client')
+const conf = require('@open-condo/config')
+
+const { DEFAULT_LOCALE } = require('@dev-api/domains/common/constants/locales')
+const { SEND_MESSAGE_MUTATION } = require('@dev-api/domains/common/gql')
+
+const DEV_AUTH_CONFIG = JSON.parse(conf['CONDO_DEV_BOT_CONFIG'] || '{}')
+const PROD_AUTH_CONFIG = JSON.parse(conf['CONDO_PROD_BOT_CONFIG'] || '{}')
+
+function _getClientRequisites (config) {
+    return {
+        endpoint: get(config, 'apiUrl', 'http://localhost:4004'),
+        authRequisites: {
+            email: get(config, 'email', 'dev-api-bot@example.com'),
+            password: get(config, 'password', 'your-secret-pass'),
+        },
+    }
+}
```

Comment:
Why not disable the use of a function without environment variables?

Reply:
Instead I would prefer to throw an error

Reply:
I guess we should think about `.env` validation outside of this PR